{"uid":"baeb9ed0a25f0a6","name":"test_case_sensitive_credentials","fullName":"tests.test_auth_login.TestAuthenticationEdgeCases#test_case_sensitive_credentials","historyId":"6003c4695a92b1c3cbe44bc16d7a5818","time":{"start":1759189483250,"stop":1759189491093,"duration":7843},"description":"Test case sensitivity of credentials.","descriptionHtml":"<p>Test case sensitivity of credentials.</p>\n","status":"failed","statusMessage":"AssertionError: Unexpected status 429 for case variation\nassert 429 in [200, 400]\n +  where 429 = <Response [429]>.status_code","statusTrace":"tests/test_auth_login.py:284: in test_case_sensitive_credentials\n    assert response.status_code in [200, 400], (\nE   AssertionError: Unexpected status 429 for case variation\nE   assert 429 in [200, 400]\nE    +  where 429 = <Response [429]>.status_code","flaky":true,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"pytestconfig","time":{"start":1759189480148,"stop":1759189480148,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"api_key","time":{"start":1759189480149,"stop":1759189480149,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"api_client","time":{"start":1759189480149,"stop":1759189480149,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"client","time":{"start":1759189480149,"stop":1759189480149,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1759189480149,"stop":1759189480149,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"login_endpoint","time":{"start":1759189480238,"stop":1759189480238,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"valid_credentials","time":{"start":1759189483249,"stop":1759189483249,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test case sensitivity of credentials.","status":"failed","statusMessage":"AssertionError: Unexpected status 429 for case variation\nassert 429 in [200, 400]\n +  where 429 = <Response [429]>.status_code","statusTrace":"tests/test_auth_login.py:284: in test_case_sensitive_credentials\n    assert response.status_code in [200, 400], (\nE   AssertionError: Unexpected status 429 for case variation\nE   assert 429 in [200, 400]\nE    +  where 429 = <Response [429]>.status_code","steps":[],"attachments":[{"uid":"3daff6c1b63878f2","name":"stdout","source":"3daff6c1b63878f2.txt","type":"text/plain","size":267}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_auth_login"},{"name":"subSuite","value":"TestAuthenticationEdgeCases"},{"name":"host","value":"runnervm3ublj"},{"name":"thread","value":"2219-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_auth_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":0,"skipped":0,"passed":1,"unknown":0,"total":5},"items":[{"uid":"cd00ceeaae874969","reportUrl":"https://sennajin.github.io/api_technical_challenge/14/#testresult/cd00ceeaae874969","status":"failed","statusDetails":"AssertionError: Unexpected status 429 for case variation\nassert 429 in [200, 400]\n +  where 429 = <Response [429]>.status_code","time":{"start":1759189278164,"stop":1759189286434,"duration":8270}},{"uid":"e7de109306ae3566","reportUrl":"https://sennajin.github.io/api_technical_challenge/13/#testresult/e7de109306ae3566","status":"failed","statusDetails":"AssertionError: Unexpected status 429 for case variation\nassert 429 in [200, 400]\n +  where 429 = <Response [429]>.status_code","time":{"start":1759189085958,"stop":1759189093660,"duration":7702}},{"uid":"49d44bb74dd5c3d5","reportUrl":"https://sennajin.github.io/api_technical_challenge/12/#testresult/49d44bb74dd5c3d5","status":"passed","time":{"start":1759188774002,"stop":1759188774268,"duration":266}},{"uid":"4e61e48725a7e1da","reportUrl":"https://sennajin.github.io/api_technical_challenge/11/#testresult/4e61e48725a7e1da","status":"failed","statusDetails":"AssertionError: Unexpected status 429 for case variation\nassert 429 in [200, 400]\n +  where 429 = <Response [429]>.status_code","time":{"start":1759188416131,"stop":1759188423837,"duration":7706}}]},"tags":["security"]},"source":"baeb9ed0a25f0a6.json","parameterValues":[]}